### ISSUE TITLE
Errors in console rather than in linter-panel

### ISSUE THREAD

**Issue by maewolfsky** [AtomLinter/linter-foodcritic#42]
After updating the base linter package to v2.1.2, I am starting to see the following errors in the console. 
Expected: see the errors and warnings in the linter-panel
```
index.js:59 [exec] Process exited with no-zero code, stdout:  Checking 1 files
x
FC003: Check whether you are running with chef server before using server-specific features: /Users/matt/git/cookbook/recipes/proc.rb:12
  (anonymous function)	@	index.js:59
  emitTwo	@	events.js:106
  emit	@	events.js:191
  maybeClose	@	internal/child_process.js:877
  (anonymous function)	@	internal/child_process.js:334
  emitOne	@	events.js:96
  emit	@	events.js:188
  _handle.close	@	net.js:493
```
```
linter-registry.js [sm]:144 [Linter] Error running foodcritic Error: Process exited with non-zero code: 3(…)
  (anonymous function) @ linter-registry.js [sm]:144
```

**Comment by maewolfsky** [comment]
Atom package versions
* linter-foodcritic version 0.5.3
* linter version 2.1.2
* linter-ui-default version 1.2.2

Atom version 1.15.0

**Comment by Arcanemagus** [comment]
Ah ha, that's actually due to updating this package to the current `atom-linter` version, and no specs here to test that things are working properly. I'll get a patch release out shortly fixing this.

**Comment by Arcanemagus** [comment]
This should be fixed in v0.5.4, thanks for reporting it!
